play-routes/play-routes-helper.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+
+OUTPUT_DIR=$(mktemp -d)
+
+PREFIX=$1
+shift
+OUTPUT_SRCJAR=$1
+shift
+ZIPPER_PATH=$1
+shift
```

Comment:
I think this would be easier as 
```suggestion
PREFIX=$1
OUTPUT_SRCJAR=$2
ZIPPER_PATH=$3
shift 3
```

Reply:
@DavidANeil thanks, that looks cleaner!